### DISCUSSION TITLE
Can i change my code into Github Pages?

### DISCUSSION THREAD

**Discussion by Malfan1** [community/community#68799] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I created Github Pages but then I found a bug, can I fix it? if so, how do I do it?

**Comment by lixavi** [comment]
Yes

**Reply by Malfan1** [reply]
How can i do it?

**Reply by vishalkrishna8** [reply]
Find the bug, edit your code to fix it, commit and push the changes to your GitHub repository, and wait for the site to update
